src/tilecache.js

```diff
@@ -330,10 +333,12 @@
          * Conversion requires tile references:
          * keep the most 'up to date' ref here. It is called and managed automatically.
          * @param {OpenSeadragon.Tile} ref
+         * @return {OpenSeadragon.CacheRecord} self reference for builder pattern
```

Comment:
In general I've been trying to move away from this pattern (on the viewer at least). If there's a good reason for it here, I suppose it's okay.

Reply:
This is due to nature of the function. It is used to directly, after creation, add property. It is just nice to be avble to run constructor().add() and have the reference of new item.

Reply:
Cool, sounds like a good reason to keep it.